src/System.Data.Common/tests/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+	  netcoreapp;
```

Comment:
Nit: indent

Reply:
tabs vs. spaces... about that... haha no just kidding ;) Fixed.